src/core/include/openvino/op/detection_output.hpp

```diff
@@ -75,5 +61,56 @@ class OPENVINO_API DetectionOutput : public Op {
     Attributes m_attrs;
 };
 }  // namespace v0
+
+namespace v8 {
+/// \brief Layer which performs non-max suppression to
+/// generate detection output using location and confidence predictions
+class OPENVINO_API DetectionOutput : public op::util::DetectionOutputBase {
+public:
+    struct Attributes : public op::util::DetectionOutputBase::AttributesBase {};
+
+    OPENVINO_OP("DetectionOutput", "opset8", op::util::DetectionOutputBase);
+    BWDCMP_RTTI_DECLARATION;
```

Comment:
You can remove `BWDCMP_RTTI_DECLARATION` from this class

Reply:
Done